app/src/pages/inside/common/accordion/accordion.jsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { AccordionTab } from './accordionTab';
+import styles from './accordion.scss';
+
+const cx = classNames.bind(styles);
+
+export const Accordion = ({
+  renderedData,
+  firstTabActive,
+  headerClassNames,
+  contentClassNames,
+}) => {
+  const [activeTabIdx, setActiveTabIdx] = useState(firstTabActive ? 0 : null);
+
+  return (
+    <div className={cx('accordion')}>
+      {renderedData &&
+        renderedData.map((section, index) => (
+          <AccordionTab
+            section={section}
+            activeTabIdx={activeTabIdx}
+            setActiveTabIdx={setActiveTabIdx}
```

Comment:
`activeTab` and `setActiveTab` will enought, it is redundant to specify what the component uses as a tab identifiers.